src/api/app/models/project.rb

```diff
@@ -9,6 +9,8 @@ class Project < ApplicationRecord
   include HasAttributes
   include MaintenanceHelper
 
+  PROJECT_TYPES = ['standard', 'maintenance', 'maintenance_incident',
```

Comment:
I would call this TYPES. So outside of project class it can be referenced viaPROJECT::TYPES  :-)